app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -411,7 +411,6 @@ declare module '@/providers/widgetRegistry' {
 
 .arrow {
   position: absolute;
-  pointer-events: none;
```

Comment:
This style was intentional, to avoid the arrow blinking when hovered in some cases. Whether or not it is visible should not affect the active drop-down area. What is the intention behind this change?

Reply:
It fixes a warning about it being defined twice. I kept the other definition because it has a comment noting that it prevents the blinking issue.